README.md

```diff
@@ -1,16 +1,21 @@
 # Islandora ISLE Interest Group
-The ISLE Interest Group consists of ISLE Maintainers, beta testers, and interested members of the community. 
+The ISLE Interest Group consists of [ISLE Maintainers](https://github.com/Islandora-Collaboration-Group/ISLE#isle-project-maintainers), beta testers, and interested members of the community. 
 
 ## Conveners
-* Benjamin Rosner (Barnard) 
-* Shaun Trujillo (MHC)
+* Shaun Trujillo, Mount Holyoke College
```

Comment:
Is this still true?  :-)   Just checking in.

Reply:
Yes, still true.